pandas/types/common.py

```diff
@@ -59,6 +59,21 @@ def is_sparse(array):
     return isinstance(array, (ABCSparseArray, ABCSparseSeries))
 
 
+# oh the troubles to reduce import time
+_spmatrix = None
+
+
+def is_scipy_sparse(array):
+    """ return if we are a scipy.sparse.spmatrix """
+    global _spmatrix
+    if _spmatrix is None:
```

Comment:
Avoided importing in a separate function as this seemed concise enough.

Am not aware of any features that would unambiguously discern scipy sparse matrices. We'd have to check for what we use later on, namely `.shape` and `.tocoo(copy: bool) -> scipy.sparse.coo_matrix`. Checking the type directly looks cleaner.